core/API/ResponseBuilder.php

```diff
@@ -164,8 +170,10 @@ private function formatExceptionMessage(Exception $exception)
 
     private function handleDataTable(DataTableInterface $datatable)
     {
-        $postProcessor = new DataTablePostProcessor($this->apiModule, $this->apiMethod, $this->request);
-        $datatable = $postProcessor->process($datatable);
+        if ($this->postProcessDataTable) {
```

Comment:
I reckon this should be a comment in the sourcecode as it's important and could be forgotten by other devs?